src/components/org/OrgChart/OrgChart.stories.tsx

```diff
@@ -154,7 +154,42 @@ const position: Position = {
                     name: 'Equinor',
                 },
                 contracts: [],
-                positions: [],
+                positions: [
```

Comment:
Positions property was added for testing right? If that's the case, we don't need it

Reply:
Removed it